<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWidget</class>
 <widget class="QWidget" name="ResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab2DPlot">
      <attribute name="title">
       <string>График</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabGraphs">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="usesScrollButtons">
          <bool>false</bool>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="tabBarAutoHide">
          <bool>false</bool>
         </property>
         <widget class="Plot2D" name="tab2DCurves">
          <attribute name="title">
           <string>2D график</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabPhase">
          <attribute name="title">
           <string>Фазовый портрет</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>9999</width>
           <height>74</height>
          </size>
         </property>
         <property name="title">
          <string>Метод</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbEulers">
            <property name="toolTip">
             <string>Синий</string>
            </property>
            <property name="text">
             <string>Метод Эйлера</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icons/icons/blue.png</normaloff>:/icons/icons/blue.png</iconset>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="cbRungeKutta">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Метод Рунге-Кутты 4-го порядка</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icons/icons/green.png</normaloff>:/icons/icons/green.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbModEulers">
            <property name="text">
             <string>Модифицированный метод Эйлера</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icons/icons/red.png</normaloff>:/icons/icons/red.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="cbAdams">
            <property name="text">
             <string>Метод Адамса-Вашфорта 4-го порядка</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icons/icons/magenta.png</normaloff>:/icons/icons/magenta.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3DPlot">
      <attribute name="title">
       <string>3D График</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabTable">
      <attribute name="title">
       <string>Таблица</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabVisualization">
      <attribute name="title">
       <string>Визуализация</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plot2D</class>
   <extends>QWidget</extends>
   <header>plot2d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
